eng/imports/HostAgnostic.props

```diff
@@ -35,13 +35,15 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub"         PrivateAssets="all" />
 
     <!-- CPS -->
-    <PackageReference Include="Microsoft.VisualStudio.ProjectSystem" />
+    <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.SDK" />
     <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.Query" />
     <PackageReference Include="Microsoft.VisualStudio.Validation" />
     <PackageReference Include="Microsoft.VisualStudio.Threading" />
     <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.SDK.Tools" />
     <PackageReference Include="System.Threading.Tasks.Dataflow" />
-
+    
+    <PackageReference Include="Microsoft.Internal.Performance.CodeMarkers.DesignTime" />
```

Comment:
Why is this `PackageReference` needed?

Reply:
The reference was required after updating the CPS/dependent packages. I don't know which package changed to require us to add this

Reply:
What error message do you get if you remove it?